src/components/RealTimeInterface/GlobeMap/LeafletMap.vue

```diff
@@ -1,43 +1,53 @@
 <script setup>
```

Comment:
Since this map has custom functionality to be a site map only, can the file be called AvailableSiteMap or something similar to be more descriptive